From 3e3d53815d197c179a0f77d7405ccaa802523371 Mon Sep 17 00:00:00 2001
From: Sandeep Singh <sandeep@freescale.com>
Date: Wed, 20 Nov 2013 16:57:40 +0530
Subject: [PATCH 13/57] powerpc/85xx: Add TDM device tree entry to T1040
 platform.

Signed-off-by: Sandeep Singh <sandeep@freescale.com>
Change-Id: I1866c9373f05c484bcb9ffeab3691c389ca5df58
Reviewed-on: http://git.am.freescale.net:8181/7231
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Poonam Aggrwal <poonam.aggrwal@freescale.com>
Reviewed-by: Jose Rivera <German.Rivera@freescale.com>
(cherry picked from commit 305cd35d3f9636d292dfaef6e6bdc885aaa43862)
Reviewed-on: http://git.am.freescale.net:8181/12336
[Original patch taken from QorIQ-SDK-V1.8-20150619-yocto.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t1040si-post.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi b/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi
index f48d5c2..6ba140e 100644
--- a/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t1040si-post.dtsi
@@ -544,6 +544,7 @@
 /include/ "qoriq-gpio-1.dtsi"
 /include/ "qoriq-gpio-2.dtsi"
 /include/ "qoriq-gpio-3.dtsi"
+/include/ "qoriq-tdm1.0.dtsi"
 /include/ "qoriq-usb2-mph-0.dtsi"
 		usb0: usb@210000 {
 			compatible = "fsl-usb2-mph-v2.5", "fsl-usb2-mph";
-- 
1.7.5.4

